conftest.py

```diff
@@ -44,7 +38,8 @@ def get_task_ids(user: str=None):
 def get_task_logs_for_id(task_id: str, lines: int=1000000):
     try:
         task_logs = subprocess.check_output([
-            'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
+            'dcos', 'task', 'log', task_id, '--lines', str(lines), 'stderr'
```

Comment:
FWIW we don't currently use stdout for anything. Only somewhat useful info is confirming all expected artifacts are fetched properly.